### ISSUE TITLE
Provide a way to chain animations

### ISSUE THREAD

**Issue by alexiscreuzot** [facebook/pop#34]
A great addition to this library would be to provide a way to chain animations by passing an array of `POPAnimation` instances to `[POPAnimator sharedAnimator]` for example.

**Comment by facebook-github-bot-1** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by freak4pc** [comment]
Anything new?

**Comment by mitchellporter** [comment]
+1 this would be awesome

**Comment by manuelescrig** [comment]
+1 Any news?

**Comment by Antanas** [comment]
+1

**Comment by fernandolicon** [comment]
+1, still waiting for any news

**Comment by RuiAAPeres** [comment]
I don't think this project, at this point in time, is being actively maintained. So @fernandolicon I wouldn't count on this feature. 😞

**Comment by fernandolicon** [comment]
@RuiAAPeres yeah, I just saw that last update was some time ago. Thanks for the heads up, I would keep using delegate methods 😕

**Comment by mxcl** [comment]
This is totally out of scope of this very well defined and well encapsulated library. If you want chaining use another library that is designed for chaining, eg. PromiseKit. Adding chaining to pop would spoil pop a little bit.

**Comment by Huang-Libo** [comment]
related pull request:
 https://github.com/facebook/pop/pull/275
